sormas-api/src/main/java/de/symeda/sormas/api/utils/EnumHelper.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * SORMAS® - Surveillance Outbreak Response Management & Analysis System
+ * Copyright © 2016-2020 Helmholtz-Zentrum für Infektionsforschung GmbH (HZI)
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package de.symeda.sormas.api.utils;
+
+public class EnumHelper {
```

Comment:
:-)
However, the use of this class for salutation still requires the developer to be aware of the mechanics behind salutation.
Would it be an option to move this code to a method like `PersonDto.renderSalutation`?
Or at least wrap it in such a method, in case this class is intended to be used for other purposes, too.

Reply:
This method is used for export where we have specific export DTO classes.

I would not create the method `PersonDto.renderSalutation` because it's not needed for now and it doesn't make sense to create unused methods. It can be created later when needed.

Reply:
A method like `PersonDto.renderSalutation` would make document templating a lot easier. A method like this could be accessed generically.
I can resolve this in other ways, but it would be the second implementation of the above resolution of the correct salutation string.

Reply:
You can call EnumHelper::toString in your implementation and there will be no double implementation. But I wouldn't mix this kind of business methods into DTOs because it can affect the serialization(e.g. converting to json in the REST api or export); it's value could accidentally be added to the serialized object.

Be careful with using this field for document generation because this field is only for german systems for now.

Reply:
Yes, already ran in an issue with `LocationDto.isEmptyLocation()` in a json serialization :-)